### DISCUSSION TITLE
How to use several commands in one custom actions command field?

### DISCUSSION THREAD

**Discussion by sojusnik** [focustimerhq/FocusTimer#674] [Q&A] [answered]
If you create a custom action, then you have a "command" field, which usually can be used with one command. But is it also possible to use two or more commands in this field?

I've tried using `|`, `&&` or `;` between two commands, but it didn't work.

Any ideas if this is possible at all?

**Comment by kamilprusko** [comment, marked as answer]
Hi :)

`|`, `&&` and `;` are shell features. The app simply starts a process with specified args as it's more lightweight. 

You could:
- execute shell command `sh -c "echo 'hello'; echo 'world'"`
- write a script and place it in `~/.local/bin`
- add an extra action through the UI

**Comment by kamilprusko** [comment]
Option "Use SubShell" lets to enter commands like in regular shell

<img width="700" height="914" alt="Screenshot From 2025-12-21 17-15-06" src="https://github.com/user-attachments/assets/20efee2c-ed50-4445-9e55-2cf5e75e8d3d" />
<img width="950" height="770" alt="Screenshot From 2025-12-21 17-15-30" src="https://github.com/user-attachments/assets/7601e53e-5d70-42c6-86bb-335eaa7cce61" />
